src/components/styles/_neon-border.less

```diff
@@ -13,6 +13,12 @@
     }
   }
 
+  [data-theme="dark"] &,
+  &.dark-mode,
+  &:hover:not(.white-btn):not(.yellow-btn) {
```

Comment:
add `:not(.white-btn):not(.yellow-btn)` to `.dark-mode` as well.

Reply:
how  do you want the background to change? I thought the current background goes well so kept as it is.

Reply:
Sorry ignore the background part, just fix the one about the yellow and white buttons.

Reply:
Umm what's the issue with yellow and white they seem to work fine in above gifs.
And  adding :not(.white-btn):not(.yellow-btn) to .dark-mode class doesnt change anything as inside
yellow and white button have explicitly defined color @black on hover.

Reply:
I have narrowed down the problem. Basically the above line says that when the dark-mode class is added, the color for all buttons should be white. So, we should add to `&.dark-mode` hover as well like `:hover:not(.white-btn):not(.yellow-btn)`

Reply:
This problem is fixed however buttons like the black button still has  issues, can you test all the buttons please that when you add the class `dark-mode` to them they behave just like when the website is changed to dark mode? Because that one is working fine.

Reply:
yes there seems to be problem in black button in dark mode its neon glow isn't visible in dark background . changes have been made in it and rest of buttons are working fine.

![captured (4)](https://user-images.githubusercontent.com/61086004/101801259-1fa24b00-3b34-11eb-874c-8b9280fa692a.gif)

Reply:
Ok it's all looking better except for the white button. When adding dark-mode class to white button, it doesn't change behavior like when the dark mode is turned on in the website.